third_party/VENDOR-LICENSE

```diff
@@ -1,7324 +0,0 @@
-
```

Comment:
This file is now empty. Please follow the below steps to fix it:
1. Run `go get -u knative.dev/test-infra` to update test-infra
2. Add the lines in `/hack/update-deps.sh` like https://github.com/knative/net-http01/pull/19/files#diff-58d4b0ee04af7710eefa27cf91255798R26-R28
3. Run `./hack-update-deps.sh` to regenerate the license file.

Reply:
Great, thanks! Done.